src/clusterfuzz/_internal/tests/core/platforms/android/settings_data/test_config.json

```diff
@@ -0,0 +1,10 @@
+{
+	"auto_build_version_to_fuzz_path": {
```

Comment:
Let's make this more generic. Like "build_paths".

Reply:
agreed. On it.